app/services/regulatory-attachments-fetcher.js

```diff
@@ -51,8 +53,14 @@ export default class RegulatoryAttachmentsFetcher extends Service {
       const templates = bindings.map((binding) => ({
         container: binding.container.value,
         title: binding.title.value,
+        reload: async (template) => {
+          const response = await fetch(
+            `${config.regulatoryStatementPreviewEndpoint}/${binding.uuid.value}`
```

Comment:
as discussed I think this should use the regular file service

Reply:
created https://binnenland.atlassian.net/browse/GN-3732